### ISSUE TITLE
Decision Tree classification

### ISSUE THREAD

**Issue by ChaoticBlack** [dscpvgcoet/Hacktoberfest2020_Galaxy_Classification#4]
Add a new file labelled "DT_sklearn.ipynb" which will classify the galaxies using a decision tree classifier. Use scikitlearn libraries as needed. 

The file should include a accuracy score and if possible a confusion matrix.

This is a good and somewhat challenging issue to sharpen your machine learning skills.
